packages/keyring-controller/src/KeyringController.ts

```diff
@@ -680,17 +730,24 @@ export class KeyringController extends BaseControllerV2<
       throw new Error('No HD keyring found.');
     }
 
+    assertHasUint8ArrayMnemonic(primaryKeyring);
+
     const seedWords = primaryKeyring.mnemonic;
     const accounts = await primaryKeyring.getAccounts();
     /* istanbul ignore if */
     if (accounts.length === 0) {
       throw new Error('Cannot verify an empty keyring.');
     }
 
+    // The HD Keyring Builder is a default keyring builder
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Nit: Is this comment necessary anymore? It doesn't seem that we are using a non-null assertion here.

Reply:
we are asserting that the return of `getKeyringBuilderForType` is non-null, so I think we still need this one